test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/MvcRazorParserTest.cs

```diff
@@ -238,7 +238,7 @@ private class TestableMvcRazorParser : MvcRazorParser
                 RazorParser parser,
                 IReadOnlyList<ChunkTree> chunkTrees,
                 IReadOnlyList<Chunk> defaultInheritedChunks)
-                : base(parser, chunkTrees, defaultInheritedChunks, typeof(ModelExpression).FullName)
+                : base(parser, chunkTrees, defaultInheritedChunks)
```

Comment:
As it relates to the MvcRazorParser ~ctor...  Why not make the make the modelExpressionTypeName parameter have a default value = null.  This will allow the other changes to remain intact (i.e. not specifying typeof(ModelExpression).FullName) and will not brake any derivatives of MvcRazorParser.  Thoughts?